src/components/reel.svelte

```diff
@@ -58,8 +56,7 @@
         class="-translate-x-1/2 -translate-y-1/2 cursor-none ring-foreground/20"
         on:click={() => {
           video.play();
-          if (trackEvent && !hasPlayedOnSession) {
-            track(trackEvent.event, trackEvent.options);
+          if (!hasPlayedOnSession) {
             hasPlayedOnSession = true;
           }
```

Comment:
I think some variables/states were created to just account for analytics events so please double check to delete them

Reply:
I double checked all the files and had left some leftover variables in 4 of them. Thank you 😄